lifelines/fitters/aalen_johansen_fitter.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import print_function
+from __future__ import division
+import numpy as np
+import pandas as pd
+import warnings
+
+from lifelines.fitters import UnivariateFitter
+from lifelines.utils import _preprocess_inputs
+from lifelines.fitters.kaplan_meier_fitter import KaplanMeierFitter
+
+class AalenJohansenFitter(UnivariateFitter):
+    """
+    Class for fitting the Aalen-Johansen estimate for the cumulative incidence function in a competing 
+    risks framework.
+    
+    AalenJohansenFitter(alpha=0.95,jitter_level=0.00001,seed=None)
+    
+    Aalen-Johansen cannot deal with tied times. We can get around this by randomy jittering the event times 
+    slightly. This will be done automatically and generate a warning.
+    """
+    def fit(self,durations,event_observed,event_ind,timeline=None,entry=None,weights=None):
+        """
+        Parameters:
+          duration: an array or pd.Series of length n -- duration of subject was observed for 
+          event_observed: an array, or pd.Series, of length n. Integer indicator of distinct events. Must be 
+             only positive integers, where 0 indicates censoring.
+          event_ind: integer -- indicator for event of interest. All other integers indicatin events in event observed
```

Comment:
I don't understand the role of this variable

Reply:
So ```event_ind``` was a shortened version of ```event_indicator``` is meant to indicate what the event code is in the ```event_observed``` column. For KaplanMeier or NelsonAalen ```F(t) = 1 - S(t)``` but since there are several different potential events you need to specify what event we are interested in ```F(t,j)``` where j corresponds to ```event_ind```

Imagine we have a dataset on time to AIDS among HIV infected patients where the outcomes are event_observed {AIDS = 1, Death = 2, Censored = 0}. For ```event_ind``` we would specify 1 since we are interested in time to AIDS and death is a competing risk. Does that make sense? Maybe another name for the input would be helpful

Reply:
@CamDavidsonPilon How about we name it ```event_code```. It may be more clear that you have to specify the indicator for the event of interest